KBKeyboardObserver/KBKeyboardObserver.m

```diff
@@ -13,7 +13,8 @@ @interface KBKeyboardObserver ()
 @property (nonatomic, assign) BOOL keyboardVisible;
 @property (nonatomic, assign) CGRect previousKeyboardBounds;
 @property (nonatomic, assign) CGRect currentKeyboardBounds;
-
+@property (nonatomic, assign) BOOL interfaceRotation;
```

Comment:
What do you thing about changing this property name to `interfaceRotates`? I think this would better fit it's boolean type.